src/main/java/com/exasol/projectkeeper/InputStreamComparator.java

```diff
@@ -0,0 +1,38 @@
+package com.exasol.projectkeeper;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+
+/**
+ * This class compares if two {@link InputStream}s are equal.
+ */
+public class InputStreamComparator {
+
+    /**
+     * Compare if the two given {@link InputStream}s are equal.
+     *
+     * @param inputStream1 first {@link InputStream} to compare
+     * @param inputStream2 second {@link InputStream} to compare
+     * @return {@code true} if streams are equal
+     * @throws IOException if reading fails
+     */
+    public boolean isEqual(final InputStream inputStream1, final InputStream inputStream2) throws IOException {
+        final byte[] buffer1 = new byte[1024 * 10];
+        final byte[] buffer2 = new byte[1024 * 10];
+        int numRead1 = 0;
+        int numRead2 = 0;
+        while (true) {
```

Comment:
It's quite hard to understand what this method does. Maybe better manes and a couple of comments  could help